app/src/main/java/org/mifos/mobile/repositories/UserAuthRepository.kt

```diff
@@ -0,0 +1,11 @@
+package org.mifos.mobile.repositories
+
+import io.reactivex.Observable
+import okhttp3.ResponseBody
+
+interface UserAuthRepository {
+
+    fun registerUser(accountNumber: String?, authenticationMode: String?, email: String?,
```

Comment:
Please reformat your code in this class an other classes you did changes.

Reply:
done reformatting based on android studio guidelines.